File: app/src/main/java/com/circles/circlesapp/chatlist/SharedFrom.java

package com.circles.circlesapp.chatlist;

import com.google.gson.annotations.SerializedName;


public class SharedFrom {

    @SerializedName("full_name")
    private String mFullName;
    @SerializedName("id")
    private Long mId;
    @SerializedName("profile_image")
    private Object mProfileImage;
    @SerializedName("share_id")
    private Long mShareId;
    @SerializedName("username")
    private String mUsername;

    public String getFullName() {
        return mFullName;
    }

    public void setFullName(String fullName) {
        mFullName = fullName;
    }

    public Long getId() {
        return mId;
    }

    public void setId(Long id) {
        mId = id;
    }

    public Object getProfileImage() {
        return mProfileImage;
    }

    public void setProfileImage(Object profileImage) {
        mProfileImage = profileImage;
    }

    public Long getShareId() {
        return mShareId;
    }

    public void setShareId(Long shareId) {
        mShareId = shareId;
    }

    public String getUsername() {
        return mUsername;
    }

    public void setUsername(String username) {
        mUsername = username;
    }

}

File: app/src/main/java/com/circles/circlesapp/retrofit/data/SharedFrom.kt
package com.circles.circlesapp.retrofit.data

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


class SharedFrom (

    @SerializedName("id")
    @Expose
    var id: Int= 0,
    @SerializedName("full_name")
    @Expose
    var fullName: String = "",
    @SerializedName("username")
    @Expose
    var username: String = "",
    @SerializedName("profile_image")
    @Expose
    var profileImage: Any = ""
)